docs/content/documentation/systems/AuxScalarKernels/tensor_mechanics/GeneralizedPlaneStrainReferenceResidual.md

```diff
@@ -0,0 +1,14 @@
+# GeneralizedPlaneStrainReferenceResidual
```

Comment:
Please put spaces among each of the words in the class name to avoid overflow problems with the longer strings